sdk/keyvault/azure-security-keyvault-keys/src/cryptography/remote_cryptography_client.cpp

```diff
@@ -1,197 +0,0 @@
-#include <azure/core/internal/http/pipeline.hpp>
-
-#include "../private/cryptography_serializers.hpp"
-#include "../private/key_constants.hpp"
-#include "../private/key_serializers.hpp"
-#include "../private/key_sign_parameters.hpp"
-#include "../private/key_verify_parameters.hpp"
-#include "../private/key_wrap_parameters.hpp"
-#include "../private/remote_cryptography_client.hpp"
-
-#include <memory>
-#include <string>
-#include <vector>
-
-using namespace Azure::Security::KeyVault::Keys;
-using namespace Azure::Security::KeyVault::Keys::Cryptography;
-using namespace Azure::Security::KeyVault::Keys::Cryptography::_detail;
-using namespace Azure::Core::Http;
-using namespace Azure::Core::Http::Policies;
-using namespace Azure::Core::Http::Policies::_internal;
-
-RemoteCryptographyClient::RemoteCryptographyClient(
-    std::string const& keyId,
-    std::shared_ptr<Core::Credentials::TokenCredential const> credential,
-    CryptographyClientOptions options)
-{
-  auto apiVersion = options.Version.ToString();
-  // Remote client is init with the URL to a key vault key.
-  KeyId = Azure::Core::Url(keyId);
-  std::vector<std::unique_ptr<HttpPolicy>> perRetrypolicies;
-  {
-    Azure::Core::Credentials::TokenRequestContext const tokenContext
-        = {{"https://vault.azure.net/.default"}};
-
-    perRetrypolicies.emplace_back(
-        std::make_unique<BearerTokenAuthenticationPolicy>(credential, tokenContext));
-  }
-
-  Pipeline = std::make_shared<Azure::Security::KeyVault::_detail::KeyVaultProtocolClient>(
-      Azure::Core::Url(keyId),
-      apiVersion,
-      Azure::Core::Http::_internal::HttpPipeline(
-          options, "KeyVault", apiVersion, std::move(perRetrypolicies), {}));
-}
-
-Azure::Response<KeyVaultKey> RemoteCryptographyClient::GetKey(
-    Azure::Core::Context const& context) const
-{
-  // The remote crypto client is created with a Key Vault key URL, hence, no path is required to get
-  // the key from the server.
-  return Pipeline->SendRequest<KeyVaultKey>(
-      context,
-      Azure::Core::Http::HttpMethod::Get,
-      [](Azure::Core::Http::RawResponse const& rawResponse) {
-        return Azure::Security::KeyVault::Keys::_detail::KeyVaultKeySerializer::
-            KeyVaultKeyDeserialize(rawResponse);
-      },
-      {});
-}
-
-EncryptResult RemoteCryptographyClient::Encrypt(
-    EncryptParameters const& parameters,
-    Azure::Core::Context const& context) const
-{
-  return Pipeline
-      ->SendRequest<EncryptResult>(
-          context,
-          Azure::Core::Http::HttpMethod::Post,
-          [&parameters]() {
-            return EncryptParametersSerializer::EncryptParametersSerialize(parameters);
-          },
-          [&parameters](Azure::Core::Http::RawResponse const& rawResponse) {
-            auto result = EncryptResultSerializer::EncryptResultDeserialize(rawResponse);
-            result.Algorithm = parameters.Algorithm;
-            return result;
-          },
-          {"encrypt"})
-      .Value;
-}
```

Comment:
I am assuming all the API implementation in `CryptographyClient` is borrowed from and matches this client. I verified this one as a spot-check (`Encrypt`), so, looks good.